packages/react/src/header/HeaderBrand.tsx

```diff
@@ -3,28 +3,148 @@ import { Box } from '../box';
 import { Flex } from '../flex';
 import { Stack } from '../stack';
 import { Text } from '../text';
-import { boxPalette, packs, tokens, useLinkComponent } from '../core';
-
-type HeaderBrandProps = {
-	badgeLabel?: string;
-	href?: string;
-	heading: string;
-	subline?: string;
-	logo?: JSX.Element;
-	size: keyof typeof logoWidthMap;
-};
+import {
+	boxPalette,
+	mapSpacing,
+	packs,
+	tokens,
+	useLinkComponent,
+} from '../core';
+import { HeaderProps } from './Header';
+
+const GAP_REM = 1.5;
+
+type HeaderBrandProps = Pick<
+	HeaderProps,
+	| 'badgeLabel'
+	| 'dividerPosition'
+	| 'heading'
+	| 'href'
+	| 'logo'
+	| 'secondHref'
+	| 'secondLogo'
+	| 'subline'
+> &
+	Required<Pick<HeaderProps, 'size'>> & { hasRightContent: boolean };
 
 export function HeaderBrand({
 	badgeLabel,
+	dividerPosition,
+	hasRightContent,
+	heading,
 	href = '/',
 	logo,
+	secondLogo,
+	secondHref,
 	size,
-	heading,
 	subline,
 }: HeaderBrandProps) {
 	const Link = useLinkComponent();
 
-	return (
+	return logo && secondLogo ? (
+		<Flex
+			css={{ overflow: 'hidden' }}
+			flexDirection={{ xs: 'column', lg: 'row' }}
+			flexWrap={{ xs: 'wrap', xl: 'nowrap' }}
+			gap={GAP_REM}
+			inline
+		>
+			<Flex
+				css={{
+					marginRight:
+						hasRightContent && dividerPosition === 'after'
+							? mapSpacing(GAP_REM)
+							: 0,
+				}} // Create a gap when there is right content
+				flexDirection={{ xs: 'column', sm: 'row' }}
+				flexShrink={0}
+				gap={GAP_REM}
+			>
+				<Flex
+					as={Link}
+					color="text"
+					css={{
+						' img, svg': { width: '100%' },
+						...packs.print.hidden,
+					}}
+					focusRingFor="keyboard"
+					href={href}
+					maxWidth={logoWidthMap[size]}
+				>
+					{logo}
+				</Flex>
+
+				{dividerPosition === 'between' && (
+					<DividingLine dividerPosition={dividerPosition} />
+				)}
+
+				<Flex
+					alignSelf={{ xs: 'start', sm: 'center' }}
+					as={Link}
+					color="text"
+					css={{
+						' img, svg': { width: '100%' },
+						...packs.print.hidden,
+					}}
+					focusRingFor="keyboard"
+					href={secondHref}
```

Comment:
`secondHref` is optional with no default, so we shouldn't render a link if it's not set

Reply:
Won't it just get a `role` of `generic`? This also follows the pattern of the `logo` + `href` prop.

Reply:
I guess, but it feels weird to render an `<a>` that has no name or id so can't be linked to. The logo one has a default of `'/'`, so that always renders a link

Reply:
Ah it does, too. Ok, I can update this.

- [ ] conditional link for second logo